sound/soc/sof/intel/hda.h

```diff
@@ -471,6 +474,7 @@ struct sof_intel_hda_stream {
 	struct hdac_ext_stream hda_stream;
 	struct sof_intel_stream stream;
 	int host_reserved; /* reserve host DMA channel */
+	bool dmi_l1_compatible;
```

Comment:
would it make sense to add a generic 'u32 flags' here and keep the value provided to hda_dsp_stream_get()?
That way you just test for the L1 compatible property when you need it, and we will never need another extension to the stream structure if we expand the flags.

Reply:
ack. done!